source/includes/_01-your-first-backend-in-10-minutes.md

```diff
@@ -0,0 +1,417 @@
+
+```text
+|- <your-project-name>
+  |- src
+    |- entities/Post.ts
+```
+Another file has been created in your project, you will need to add the implementation of its projection:
+
+> Projection
+
+```typescript
+@Reduces(PostCreated)
+public static projectPostCreated(event: PostCreated, currentPost?: Post): Post {
+return new Post(event.postId, event.title, event.content, event.author)
+}
+```
+
+In the future, we may want to *project* events for this `Post` entity that require retrieving current `Post` values. In that case we would need to make use of `currentPost` argument. 
+
+> `Post` entity
+
+```typescript
+@Entity
+export class Post {
+  public constructor(
+    public id: UUID,
+    readonly title: string,
+    readonly content: string,
+    readonly author: string,
+  ) {}
+
+  @Reduces(PostCreated)
+  public static projectPostCreated(event: PostCreated, currentPost?: Post): Post {
+    return new Post(event.postId, event.title, event.content, event.author)
+  }
+}
+```
+The full code for the entity can be seen on the right.
+
+## 5. First read model
+> New read model
+
+```bash
+boost new:read-model PostReadModel --fields title:string content:string author:string --projects Post
+```
+Almost everything is set-up. We just need to provide a way to view the `Posts` of our blog. For that, we will create a `read model`.
+
+```text
+|- <your-project-name>
+  |- src
+    |- read-models/PostReadModel.ts
+```
+Once the read-model code has been generated, we will also need to define a few things:
+
+- Who can read the `Posts`
+- How the data is manipulated before returning it
+
+```typescript
+@ReadModel({
+  authorize: 'all'// Specify authorized roles here. Use 'all' to authorize anyone
+})
+```
+To make it easy, we will allow anyone to read it:
+
+```typescript
+@Projects(Post, "id")
+  public static projectPost(entity: Post, currentPostReadModel?: PostReadModel): PostReadModel {
+    return new PostReadModel(entity.id, entity.title, entity.content, entity.author)
+  }
+```
+and we will project the whole entity
+
+> `PostReadModel` read model
+
+```typescript
+@ReadModel({
+  authorize: 'all'// Specify authorized roles here. Use 'all' to authorize anyone
+})
+export class PostReadModel {
+  public constructor(
+    public id: UUID,
+    readonly title: string,
+    readonly content: string,
+    readonly author: string,
+  ) {}
+
+  @Projects(Post, "id")
+  public static projectPost(entity: Post, currentPostReadModel?: PostReadModel): PostReadModel {
+    return new PostReadModel(entity.id, entity.title, entity.content, entity.author)
+  }
+}
+```
+The read model should look like the code on the right:
+
+## 6. Deployment
+```bash
+boost deploy -e production
+```
+Everything we need for a basic project is set. It is time to deploy it:
+
+It will take a couple of minutes to deploy all the resources.
+
+> GraphQL endpoint
+
+```text
+https://<api-gateway-id>.execute-api.us-east-1.amazonaws.com/production/graphql
```

Comment:
Perhaps this? Users could have different regions configured, also they do not need to know what's API GW

```suggestion
> Example GraphQL endpoint

```text
https://<some random number>.execute-api.us-east-1.amazonaws.com/production/graphql
```